vercel.json

```diff
@@ -1,5 +1,60 @@
 {
   "redirects": [
+    {
+      "source": "/graphql-js",
+      "destination": "https://graphql-js.org",
+      "permanent": true
+    },
+    {
+      "source": "/graphql-js/getting-started",
+      "destination": "https://graphql-js.org",
+      "permanent": true
+    },
+    {
+      "source": "/graphql-js/graphql",
+      "destination": "https://graphql-js.org/api-v16/graphql",
+      "permanent": true
+    },
+    {
+      "source": "/graphql-js/error",
+      "destination": "https://graphql-js.org/api-v16/error",
+      "permanent": true
+    },
+    {
+      "source": "/graphql-js/execution",
+      "destination": "https://graphql-js.org/api-v16/execution",
+      "permanent": true
+    },
+    {
+      "source": "/graphql-js/language",
+      "destination": "https://graphql-js.org/api-v16/language",
+      "permanent": true
+    },
+    {
+      "source": "/graphql-js/type",
+      "destination": "https://graphql-js.org/api-v16/type",
+      "permanent": true
+    },
+    {
+      "source": "/graphql-js/utilities",
+      "destination": "https://graphql-js.org/api-v16/utilities",
+      "permanent": true
+    },
+    {
+      "source": "/graphql-js/validation",
+      "destination": "https://graphql-js.org/api-v16/validation",
+      "permanent": true
+    },
+    {
+      "source": "/graphql-js/graphql-http",
+      "destination": "https://graphql-js.org/api-v16/graphql-http",
+      "permanent": true
+    },
```

Comment:
Is this the right place for graphql-http documentation?

Reply:
I'm not sure honestly, I'm not sure it belongs in graphql-js even 😅